<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInformationPageUI</class>
 <widget class="QWidget" name="UserInformationPageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>30</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INFORMATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Day of birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Role:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Phone Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 0px;
	border-radius: 3px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 0px;
	border-radius: 3px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="role">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 0px;
	border-radius: 3px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dayOfBirth">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>yyyy/mm/dd</string>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 0px;
	border-radius: 3px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumber">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 0px;
	border-radius: 3px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 0px;
	border-radius: 3px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
          background-color: rgb(45, 18, 153);
          color: #fff; 
          border-radius: 6px;
      } 
      QPushButton:hover { 
          background-color: rgb(55, 45, 135);
      }
      QPushButton:hover:!pressed{
          border: 1.5px solid rgb(50, 45, 50);
      }</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>If something went wrong, &lt;a href=&quot;change_password&quot;&gt; change password&lt;/a&gt; or &lt;a href = &quot;report&quot;&gt;report with admin!&lt;/a&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_log">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
